src/output-json-ike.h

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2015 Open Information Security Foundation
+/* Copyright (C) 2015-2020 Open Information Security Foundation
```

Comment:
Do we need to change this file ?

Reply:
Probably not in this commit. Frank just added himself as additional author, but in fact only removed the 'v2' suffix from the functions in the previous commit. I can simply remove that if you want.

Reply:
Yes, let's minimize the changes

Reply:
Done